build_scripts/es_setup/es_setup.sh

```diff
@@ -41,7 +41,25 @@ chmod 777 -R /data
 # 5. Start Elasticsearch
 sudo /etc/init.d/elasticsearch start
 
-# 6. Add the following lines to /etc/sudoers file
+# 6. Default setup for searchguard plugin
+sudo /etc/init.d/elasticsearch stop
+sudo cp /vagrant/build_scripts/es_setup/search-guard-5.zip /tmp/
+cd /tmp/
+sudo unzip search-guard-5.zip
+sudo mv search-guard-5 /usr/share/elasticsearch/plugins/
+#sudo unzip /usr/share/elasticsearch/plugins/search-guard-5.zip
+#sudo rm -rf search-guard-5.zip
+sudo cp /usr/share/elasticsearch/plugins/search-guard-5/tools/install_configuration.sh /usr/share/elasticsearch/plugins/search-guard-5/tools/install_configuration.sh.orig
+sudo cp /vagrant/build_scripts/es_setup/install_demo_configuration.sh /usr/share/elasticsearch/plugins/search-guard-5/tools/
+sudo chmod +x /usr/share/elasticsearch/plugins/search-guard-5/tools/install_configuration.sh
+cd /usr/share/elasticsearch/plugins/search-guard-5/tools/
+sudo ./install_configuration.sh -y
+sudo /etc/init.d/elasticsearch start
+sudo chmod +x /usr/share/elasticsearch/plugins/search-guard-5/tools/sgadmin.sh
+# sudo /usr/share/elasticsearch/plugins/search-guard-5/tools/sgadmin.sh -cd /usr/share/elasticsearch/plugins/search-guard-5/sgconfig -cn openrecords_v2_0_development -ks /etc/elasticsearch/kirk.jks -ts /etc/elasticsearch/truststore.jks -nhnv -icl
```

Comment:
Your comment below refers to this line?

Why is this the case?

Reply:
I tried using a vagrant reload right before the last line and it didn't work. Running manually after the build scripts have finished is the only way I found for it to work.